tutorials/vpn-interop-guides/interop-guide-template/index.md

```diff
@@ -0,0 +1,1003 @@
+---
+title: Interoperability Guide Template for Cloud VPN
+description: Enable third-party vendors to create an Interop guide for Cloud VPN.
+author: ashishverm
+tags: Cloud Networking, Cloud VPN
+date_published: 2018-05-18
+---
+
+# Google Cloud VPN Interop Guide
+
+Using Cloud VPN with <vendor name><product name>Learn how to build site-to-site IPSEC VPN between Cloud VPN and `<vendor name/>/<product name>`.
```

Comment:
@jmdobry Style question. We typically use the tick marks for code formatting. For this template, should we use this formatting for text that should replaced, or should we just escape the angle brackets without the code formatting?

Reply:
@JimTravis can you take a look at this question about the use of code formatting to indicate replacement text? thanks!